helm-charts/aztec-network/templates/anvil.deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
should this be stateful?

Reply:
Yeah it probably should/will. We might be able to test L1 censorship by killing the anvil pod, and when it comes back it shouldn't start from scratch.